test/config/auth_providers_config.exs

```diff
@@ -0,0 +1,7 @@
+import Config
```

Comment:
What about using test.exs for this vs a new file, and would avoid having to read an extra file?

Reply:
I can set this dummy config in `test.exs` to test the happy path, but in order to test the case when any of these env vars are missing I would need to read another config in a separate file.
Another option could be to update the env vars inside the test case (using `Application.put_env/3`) and ensure that on exit the original values are reset. Wdyt?

Reply:
I checked and works fine for both successful and failure scenarios. Just pushed the updates, thanks!

Reply:
Awesome nice improvement!